App/Talent.App.WebApp/wwwroot/js/react/ReactScripts/Profile/PhotoUpload.jsx

```diff
@@ -1,20 +1,178 @@
-﻿/* Photo upload section */
-import React, { Component } from 'react';
+﻿import React, { Component } from 'react';
 import Cookies from 'js-cookie';
+import { Button, Form, Grid, Icon, Input, Image, Label, Dropdown } from 'semantic-ui-react';
+import AWS from 'aws-sdk';
+import '@babel/polyfill';
 
 export default class PhotoUpload extends Component {
 
     constructor(props) {
         super(props);
 
-       
+        this.state = {
+            selectedFile: null,
+            selectedFileUrl: "",
+            
+        }
+
+        this.fileChange = this.fileChange.bind(this);
+        this.uploadFile = this.uploadFile.bind(this);
+        this.renderPlaceholder = this.renderPlaceholder.bind(this);
+        this.renderThumbnail = this.renderThumbnail.bind(this);
     };
 
+    fileChange(event) {
+       
+        
+        const acceptedExtensions = ['jpg', 'png', 'gif', 'jpeg'];
+        const file = event.target.files[0];
+
+        if (file) {
+            const fileExtension = file.name.split('.').pop().toLowerCase();
+            
+            if (!acceptedExtensions.includes(fileExtension)) {
+                alert("Please only select an image file with extensions 'jpg', 'png', 'gif', 'jpeg'");
+                return;
+            }
+
+            this.setState({
+                selectedFile: file,
+                selectedFileUrl: URL.createObjectURL(file)
+            });
+        }
+
+
+    }
 
     
 
-    render() {
+    async uploadFile() {
+        
+        
+
+        const awsAccessKeyId = 'AKIAVCTCWZ4Q7DK6I2H7';
+        const awsSecretAccessKey = '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';
```

Comment:
This is fine for now, but bear in mind that we don't have to expose any IDs in our code specially for any access. There are different ways to handle this like for example using environment variables. You don't need to implement it in this task but please take note of it and you can do some research about it